.test-infra/jenkins/PrecommitJobBuilder.groovy

```diff
@@ -30,7 +30,7 @@ class PrecommitJobBuilder {
   String gradleTask
 
   /** Overall job timeout. */
-  int timeoutMins = 30
```

Comment:
Please undo this change, and only set 120 for Java precommits (with a comment to reset to 30 once BEAM-5035 is fixed).